kerlescan/exceptions.py

```diff
@@ -53,3 +53,12 @@ def __init__(self, message):
         """
         super(RuntimeError, self).__init__()
         self.message = message
+
+
+class IllegalHttpMethodError(ValueError):
+    def __init__(self, message):
+        """
+        Raise this exception if we detect illegal HTTP method
+        """
+        super(IllegalHttpMethodError, self).__init__()
```

Comment:
Nit: In Python 2.7+ you can call `super().__init__()`

Also if the message isn't passed to `ValueError`, then calling `str()` would give an empty string.

Reply:
I am aware of that `str()` thing.

Thanks for that `super()` notation.